addons/resource.language.en_gb/resources/strings.po

```diff
@@ -24700,3 +24700,14 @@ msgid "50th Anniversary Edition"
 msgstr ""
 
 # 40000 to 40800 are reserved for Video Versions feature
+
+#: system/settings/settings.xml
+msgctxt "#40801"
+msgid "Confirm file deletion"
```

Comment:
This not only asks to delete files but you involved it in to profile management
to me its a bit inconsistent